src/fromager/commands/download_sequence.py

```diff
@@ -59,31 +60,26 @@ def download_sequence(
         build_order = json.load(f)
 
     def download_one(entry: dict[str, typing.Any]):
+        req = Requirement(f"{entry['dist']}=={entry['version']}")
+
         if entry["prebuilt"]:
-            logger.info(f"{entry['dist']}: uses a pre-built wheel, skipping")
+            if include_wheels:
+                wheels.download_wheel(req, entry["source_url"], wkctx.wheels_downloads)
```

Comment:
Will the URL in the entry of a build-order file point to a wheel?

Reply:
yes it should https://github.com/python-wheel-build/fromager/blob/main/src/fromager/sdist.py#L140

Reply:
nice, ok